/**
 * This package contains exception under the authentication package.
 * Subpackages of "exception" (such as this one) follow the convention of mirroring the project structure
 * of the classes that throw each exception
 * @since 0.0.1-SNAPSHOT
 * @author xdavide9
 */
package com.xdavide9.sso.exception.authentication;
